subsys/bluetooth/controller/ll_sw/ull_sync.c

```diff
@@ -654,6 +658,25 @@ void ull_sync_setup(struct ll_scan_set *scan, struct ll_scan_aux_set *aux,
 		  (ret == TICKER_STATUS_BUSY));
 }
 
+void ull_sync_setup_cleanup(struct ll_scan_set *scan)
```

Comment:
The comment: `/* Reset the sync context allocated to scan contexts */` describes the function better than its name.
```suggestion
void ull_sync_reset_in_scan_ctx(struct ll_scan_set *scan)
```

Reply:
The naming is inline with `*_cleanup` used like in `ull_central_cleanup` to remove connection context associated with scan context during connection establishment.

Reply:
`void ull_central_cleanup(struct node_rx_hdr *rx_free)` does really cleanup of a `rx_free`. The `ull_sync_setup_cleanup` just assigns `NULL` to `sync` member. It does not release the `sync`. The automatic release of the sync is something that I would expect when see call to the function.

It could be misleading when someone reads the code. You have to remember the content of the function or put limited trust in what you may expect until you check the implementation.

Reply:
I have renamed it to `ull_sync_setup_complete`. `reset` is associated with HCI or HAL reset implementations.